app/src/common/shared/com/igalia/wolvic/ui/widgets/settings/DeveloperOptionsView.java

```diff
@@ -219,9 +244,33 @@ private void setLocalAddon(boolean value, boolean doApply) {
         }
     }
 
+    private void setUseWolvicUA(boolean value, boolean doApply) {
+        mBinding.wolvicUserAgentSwitch.setOnCheckedChangeListener(null);
+        mBinding.wolvicUserAgentSwitch.setValue(value, false);
+        mBinding.wolvicUserAgentSwitch.setOnCheckedChangeListener(mUseWolvicUAListener);
+
+        if (doApply) {
+            SettingsStore.getInstance(getContext()).setUseWolvicUA(value);
+        }
+    }
+
+    private void setUaMode(int checkId, boolean doApply) {
+        mBinding.uaRadio.setOnCheckedChangeListener(null);
+        mBinding.uaRadio.setChecked(checkId, doApply);
+        mBinding.uaRadio.setOnCheckedChangeListener(mUaModeListener);
+
+        SettingsStore.getInstance(getContext()).setUaMode((Integer) mBinding.uaRadio.getValueForId(checkId));
+    }
+
+    @Override
+    public Point getDimensions() {
+        return new Point(WidgetPlacement.dpDimension(getContext(), R.dimen.settings_dialog_width),
+                WidgetPlacement.dpDimension(getContext(), R.dimen.developer_options_height));
+    }
+
     @Override
     protected SettingViewType getType() {
-        return SettingViewType.LANGUAGE_VOICE;
+        return SettingViewType.DEVELOPER;
```

Comment:
Same comment as above.

Reply:
This is a little issue that I saw when changing the dialog, it seems too small for its own PR but I can open one if needed.